app/models/form/sales/pages/person_known.rb

```diff
@@ -0,0 +1,22 @@
+class Form::Sales::Pages::PersonKnown < Form::Sales::Pages::Person
+  def initialize(id, hsh, subsection, person_index:)
+    super
+    @header_partial = "#{id}_page"
+    @header = ""
+    @description = ""
+    @subsection = subsection
+    @depends_on = page_depends_on
+  end
+
+  def questions
+    @questions ||= [
+      Form::Sales::Questions::PersonKnown.new(field_for_person("details_known_"), nil, self, person_index: @person_index),
+    ]
+  end
+
+  def page_depends_on
+    return (@person_index..4).map { |index| { "hholdcount" => index } } if @person_index == 1
```

Comment:
slightly odd that in other cases `@person_index` is 2-5 or 3-6 depending on if it's a joint purchase but here it's 1-4, not sure if there's a clearer way of doing this?